test/e2e/disruption.go

```diff
@@ -22,13 +22,12 @@ import (
 
 	. "github.com/onsi/ginkgo"
 	. "github.com/onsi/gomega"
-	"k8s.io/client-go/1.5/kubernetes"
-	apiapi "k8s.io/client-go/1.5/pkg/api"
-	"k8s.io/client-go/1.5/pkg/api/unversioned"
-	api "k8s.io/client-go/1.5/pkg/api/v1"
-	extensions "k8s.io/client-go/1.5/pkg/apis/extensions/v1beta1"
-	policy "k8s.io/client-go/1.5/pkg/apis/policy/v1alpha1"
-	"k8s.io/client-go/1.5/pkg/util/intstr"
+	"k8s.io/client-go/kubernetes"
```

Comment:
I guess my test was the only one that was using 1.5 directly.  This rename looks easy, but what if some test *wants* to use an old API.  For example, suppose we have some bug that was tickled specifically by, say, the 1.3 client.  Will we be able to write such tests?